### DISCUSSION TITLE
Support for activating conda environments

### DISCUSSION THREAD

**Discussion by aslowwriter** [jdx/mise#1514] [Ideas]
Hi folks 👋 First of all thanks for the great work on mise! 

A lot of my work is in conda environments for python, and I couldn't find a way to for example activate a conda environment under mise let alone create one. If there is one I did not find it and perhaps this would be good to add to the documentation. If there isn't then I'd love there to be one! support for interacting with conda environments directly would be great but I can imagine that that's a bit much. 

In any case, thanks for everythign!

**Comment by astrojuanlu** [comment]
Came here to open the same issue! 😄

**Comment by jdx** [comment]
this is something that should go into a conda plugin. You can look at [mise-poetry](https://github.com/mise-plugins/mise-poetry/blob/main/bin/exec-env) to get an idea of how to do this.

**Reply by jacobprudhomme** [reply]
Not to necrobump, but I would be interested in starting work on this. Is this still a good place to look at? Or has that changed now that aqua is the preferred way to add tools to mise (asking because I'm not familiar with how aqua works and thus which tools are suitable to be added to the registry or not)?

**Reply by jdx** [reply]
aqua is not capable of something like this, it requires an asdf or vfox plugin
